examples/next-prisma-starter/src/pages/_app.tsx

```diff
@@ -19,7 +19,7 @@ function getBaseUrl() {
   }
   // // reference for vercel.com
   // if (process.env.VERCEL_URL) {
-  //   return process.env.VERCEL_URL;
+  //   return `https://${process.env.VERCEL_URL}`;
```

Comment:
Have you verified that vercel doesn't already include https?

Reply:
ya, had an "only allows absolute urls" error on vercel and headed over to their docs: https://vercel.com/docs/environment-variables#system-environment-variables

```
VERCEL_URL | The URL of the deployment. Example: my-site-7q03y4pi5.vercel.app.
```

Reply:
if we want this to be nice we could use sth like url.parse

Reply:
Awesome, will double check but will merge this for now, thank you so much! 🙏